### ISSUE TITLE
Send reports to Lighthouse CI Server

### ISSUE THREAD

**Issue by manuman94** [GoogleChrome/lighthouse#13495]
**Feature request summary**
Send reports generated programmatically by Lighthouse to a remote instance of Lighthouse CI Server.

**What is the motivation or use case for changing this?**
We are working with [playwright-lighthouse](https://github.com/abhinaba-ghosh/playwright-lighthouse) and we would like to throw the generated reports to a Lighthouse CI server. The aim is to have them published and to compare them between commits.

**How is this beneficial to Lighthouse?**
Lighthouse already generates HTML reports based on the test execution. It would be better to publish them and take advantage of the awesome Lighthouse CI Server features.

**Comment by manuman94** [comment]
I finally got my lighthouse project to generate reports and upload them to a Lighthouse Server CI.

Here is the project I made: https://github.com/manuman94/playwright-lighthouse-ci-demo

The only thing I would need is to execute the Lighthouse process to the current page HTML without reloading (to the page "snapshot").

I saw that there is already an issue open working on this, I will follow that topic: https://github.com/GoogleChrome/lighthouse/issues/11313

Greetings.
